EC-84: Revise Configuration, Customers, eContent, Login and Logging (#5)
ezsystems/commerce-documentation: docs/guide/customers/managing_delivery_addresses.md

## Read an existing delivery address from NAV

> In NAV, the address code is needed in addition to the party identification in order to fetch a delivery address.

Reviewer: ```suggestion
In NAV, to fetch a delivery address, the address code is required apart from the party identification.
```

> In NAV, the address code is needed in addition to the party identification in order to fetch a delivery address.

Reviewer 2: ```suggestion
In NAV, the address code and party identification are required in order to fetch a delivery address.
```

## Update an existing delivery address in NAV

Complete data must be sent (changed and unchanged fields) to change the address,
together with the Key value of the last read data. The Key is a consistency check for the update.

> If the data in NAV changed since it was fetched the last time, the Keys of the update request and in NAV would mismatch and the request fails.

Reviewer: ```suggestion
If the data in NAV changed since it was fetched the last time, the Key of the update request does not match the Key in NAV and the request fails.
```

> In that case, the data must be fetched again and the current changes must be merged into the new data from NAV.

Reviewer: ```suggestion
In that case, the data must be fetched again and current changes must be merged into the new data from NAV.
```

> This might need user interaction (several HTTP requests).

Reviewer: ```suggestion
Fetching data might require user interaction (several HTTP requests).
```

> After that, the merged data must be sent again in an update request with the new Key.

Reviewer: ```suggestion
The merged data must then be sent again in an update request with a new Key.
```
